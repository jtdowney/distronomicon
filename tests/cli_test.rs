use assert_cmd::Command;
use insta::assert_snapshot;

#[test]
fn test_help_shows_subcommands() {
    let mut cmd = Command::cargo_bin("distronomicon").unwrap();
    let output = cmd.arg("--help").output().unwrap();

    let stdout = String::from_utf8_lossy(&output.stdout);

    assert_snapshot!(stdout);
}
