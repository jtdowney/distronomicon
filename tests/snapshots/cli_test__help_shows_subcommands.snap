---
source: tests/cli_test.rs
expression: stdout
---
Usage: distronomicon [OPTIONS] <COMMAND>

Commands:
  check    Check for updates without installing
  update   Update to the latest release
  version  Show currently installed version
  help     Print this message or the help of the given subcommand(s)

Options:
  -v, --verbose...  
  -h, --help        Print help
